TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,5 +6,74 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
     {
+        private char[,] space; //Board
+        private int gridNumber = 0; //Board
+
+        private void ChooseBoard() //Board
+        {
+            Console.WriteLine("\nThis grid is user defined.");
+            Console.WriteLine("ie.Inputing 3 will result in a grid 3 rows by 3 columns.");
+            gridNumber = GetUserInput(3, 1000);
+            space = new char[gridNumber, gridNumber];
+        }
+
+        private void InitializeBoard() //Board
+        {
+            for (int row = 0; row < gridNumber; row++)
+            {
+                for (int col = 0; col < gridNumber; col++)
+                {
+                    space[row, col] = ' ';
+                }
+            }
+        }
+
+        private void DrawBoard()  //Board
+        {
+            Console.Clear();
+            Console.Write("    ");
+            for (int col = 0; col < gridNumber; col++)
+            {
+                Console.Write(col + "   ");
+            }
+
+            Console.WriteLine();
+
+            for (int row = 0; row < gridNumber; row++)
+            {
+                Console.Write(row + " | ");
+                for (int col = 0; col < gridNumber; col++)
+                {
+                    Console.Write(space[row, col]);
+                    Console.Write(" | ");
+                }
+                Console.WriteLine();
+            }
+        }
+        private void GetUserInput()
```

Comment:
This function would fit better elsewhere. My goal is for the board to be purely a representation of the moves that have been made in the game.